Bezier.cpp

```diff
@@ -9,4 +9,32 @@ Bezier::Bezier(const Vector2& p0, const Vector2& p1, const Vector2& p2, const Ve
 void Bezier::draw(SDL_Renderer *renderer) const
 {
 	// TODO
+
+	float t = 0.0f;
+	float tt = 0.0f;
```

Comment:
`t` and `tt` aren't very descriptive variable names -- `t` is OK since it's used in the maths from the worksheet, but better names would be something like `tCurrent` and `tNext` for example.